### ISSUE TITLE
Add test for opening BDVN5S3

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-io#26]
@KateMoreva the current `main` branch contains this code:

```
public class OpenBDVN5S3XML {
    public static void main(String[] args) throws SpimDataException {
        SpimData spimData = new XmlIoSpimData().load("src/test/resources/prospr-myosin-n5.xml");
        BdvFunctions.show(spimData);
    }
}
```

What about adding a `@Test` annotation to this to thereby also cover the bdv.n5.s3 code?

**Comment by KateMoreva** [comment]
To make a proper test adding an annotation is not enough. I will take a look and make a proper test here as well.

**Comment by constantinpape** [comment]
Closed in favor of #87
